recipes-graphics/mesa/mesa_%.bbappend

```diff
@@ -2,5 +2,9 @@
 # With oe-core commit 8509e2e1a87578882b71948ccef3b50ccf1228b3 dri3 is set
 # as default. To state out clearly that Raspi needs dri3 and to avoid surprises
 # in case oe-core changes this default, we set dri3 explicitly.
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
You should add this to the top of the file. Inserting after the comment above adds confusion.